tools/cloud-build/images/cluster-toolkit-dockerfile/README.md

```diff
@@ -43,7 +47,7 @@ docker run -v ~/.config/gcloud/:/root/.config/gcloud -v $(pwd):/out <image_name>
 
 This command runs the Cluster Toolkit Docker image and allows the `gcluster` binary to access your Google Cloud credentials and local files. Here's a breakdown:
 
-* `-v ~/.config/gcloud/:/root/.config/gcloud`: This argument mounts your local Google Cloud configuration directory `(~/.config/gcloud)` to the `/root/.config/gcloud` directory inside the container. This allows the `gcluster` binary to access your credentials and interact with Google Cloud resources when needed.
+* `-v ~/.config/gcloud/:/root/.config/gcloud`: This argument mounts your local Google Cloud configuration directory `(~/.config/gcloud)` to the `/root/.config/gcloud` directory inside the container. This allows the `gcluster` binary to access your credentials and interact with Google Cloud resources when needed. If not already done, running `gcloud auth application-default login` should generate the Application Default Credentials in the `~/.config/gcloud/` directory on your local machine.
```

Comment:
I'd suggest putting the parentheses outside the code block so it's clear that they're not part of the string.

```suggestion
* `-v ~/.config/gcloud/:/root/.config/gcloud`: This argument mounts your local Google Cloud configuration directory (i.e. `~/.config/gcloud`) to the `/root/.config/gcloud` directory inside the container. This allows the `gcluster` binary to access your credentials and interact with Google Cloud resources when needed. If not already done, running `gcloud auth application-default login` should generate the Application Default Credentials in the `~/.config/gcloud/` directory on your local machine.
```